base-theme/assets/index.ts

```diff
@@ -37,3 +37,12 @@ $(document).ready(() => {
 
   initSentry()
 })
+
+// &nbsp; causes text to overflow and it doesn't wrap inside a div
+// not sure if this is a good approach but I couldn't ANY Hugo template function
+// with which we can get rid of &nbsp;
+// not wrapping this code in function for better efficiency and to avoid glitch for user
+const elements = document.getElementsByClassName("remove-nbsp")
```

Comment:
This seems more like it's a problem upstream with `ocw-to-hugo`.  I checked out the `ocw-to-hugo` output for the course indicated in the issue, and the `title` property on the indicated page is this:

`title: "Review\_of\_October\_Course\_Requirements\_and\_Deliverables"`

For some reason, `\_` is being inserted into the title where there should just be a space.  This is likely creating the HTML character entity `&nbsp;` and it should probably be addressed there and not after the fact here in the theme.

Reply:
@gumaerc Yeah, you're right, this should not be addressed here. 
Thank you for looking into it.

I was also wondering that from where `\_` is being appended, it seems to be in titles and some descriptions too, not sure if it occurs somewhere else too. 
Do you have any idea from where this `\_` is coming from in the first place? If not, then I can try finding it out. What do you suggest? 

For now, if you have addressed this in `ocw-to-hugo` via regex then I'll remove the custom js solution from here and keep the expand/collapse as it is. Correct?

Reply:
There's 20 years of data coming out of the legacy OCW system. If you give me an example course and file name (or URL) I can look into it and speculate where those unicode characters are coming from, but it probably doesn't matter. 

If it's easy to fix them up with a regex in the migration, great. If not, we can ask the OCW team to fix them when they see them. It might be helpful to know how many of these there are to inform the decision of automation vs. manual.

Reply:
@pdpinch Yeah, let's fix them up via regex, which I think @gumaerc has already done for titles. 
Although after seeing a bit more into it, it looks like this issue is in multiple places and courses. For [example](https://github.mit.edu/raw/ocw-content-rc/21g.049-spring-2017/main/content/resources/21g-049s17-th.md?token=AAAFYX4CZ22R46M6V7YFQFLB7I34A). So I think it will be really hard to identify and do it manually so I guess we are good with fixing via regex.